### ISSUE TITLE
Add threshold for abstractive summaries.

### ISSUE THREAD

**Issue by KennethEnevoldsen** [Danish-summarisation/DanSum#3]
E.g.
```
python train.py --learning_rate 0.5 --abstractive_threshold 0.4
```

test set should still include abstractive (as well as mixed and extractive). Validation set should be abstractive.
